src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useMemo } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const TabsPage: React.FC = () => {
+  const { tabId } = useParams();
+
+  const selectedTab = useMemo(() => {
+    return tabs.find(currentTab => currentTab.id === tabId);
+  }, [tabId]);
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(currentTab => (
+            <li
+              data-cy="Tab"
+              key={currentTab.id}
+              className={classNames({
+                'is-active': tabId === currentTab.id,
+              })}
+            >
+              <Link to={`../${currentTab.id}`}>
+                {currentTab.title}
+              </Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      {selectedTab ? (
+        <div className="block" data-cy="TabContent">
+          {selectedTab?.content}
+        </div>
+      ) : (
+        <div className="block" data-cy="TabContent">
+          Please select a tab
+        </div>
+      )}
```

Comment:
```suggestion
        <div className="block" data-cy="TabContent">
          {selectedTab ? selectedTab?.content :  Please select a tab}
        </div>
```
we can set only text in ternary